data/roles.js

```diff
@@ -629,157 +648,163 @@ const roleData = {
+                "Visits a player to appears as their role.",
+                "Learns chosen player's role."
             ],
         },
         "Prosecutor": {
             alignment: "Mafia",
             description: [
-                "Votes weight is 2."
+                "Vote weight is worth 2 players votes in village meeting."
             ],
         },
         "Fabricator": {
             alignment: "Mafia",
             description: [
-                "Once per night can give out an ill-fated version of many items."
+                "Gives out a fabricated item once per night.",
+                "Fabricated Guns and Knives will backfire against the player who used them.",
+                "Fabricated Armor, Crystals and Snowballs will be ineffective."
             ],
         },
         "Heartbreaker": {
             alignment: "Mafia",
             description: [
-                "Once per game can fall in love with another player.",
-                "If Heartbreaker dies, both players will die.",
+                "Fall in love with another player once per game.",
+                "Both players will die if Heartbreaker dies.",
             ],
         },
         "Yakuza": {
             alignment: "Mafia",
             description: [
-                "Can choose to sacrifice themself to convert one player to Mafioso.",
+                "Chooses to sacrifice self once per game to convert another player to Mafioso.",
             ],
         },
         "Necromancer": {
             alignment: "Mafia",
             description: [
-                "Once per game, visits one dead person during the night.",
-                "That person will be resurrected.",
+                "Visits a dead player during the night once per game.",
+                "That player will be resurrected the following day.",
+                "If players identity was revealed upon death, they will remain revealed when ressurected."
             ],
             graveyardParticipation: "all",
         },
         "Mummy": {
             alignment: "Mafia",
             description: [
-                "Everyone who visits the mummy while the mummy is dead dies.",
+                "Everyone who visits the mummy while the mummy is dead, dies.",
             ],
         },
         "Poltergeist": {
             alignment: "Mafia",
             description: [
-                "Once dead, may visit one person a night and roleblock them.",
+                "Once dead, visits one player each night and roleblock them.",
            ],
            graveyardParticipation: "self",
         },
         "Informant": {
             alignment: "Mafia",
             description: [
-                "Can pick a person every night.",
-                "Any system messages that person may have got are given to the informant as well"
+                "Chooses a player each night and views any system messages they recieve the following day."
```

Comment:
Wording? Maybe we should change it to "views any reports" instead of "system messages"? It feels more natural for the game, in my opinion.